sbt_common/internal_model/src/test/scala/uk/ac/wellcome/models/work/test/util/WorksGenerators.scala

```diff
@@ -3,7 +3,7 @@ package uk.ac.wellcome.models.work.test.util
 import uk.ac.wellcome.models.work.internal._
 
 trait WorksGenerators extends ItemsGenerators {
-  private def createTitle: String = randomAlphanumeric(length = 100)
+  private def createRandomTitle: String = randomAlphanumeric(length = 100)
```

Comment:
nit: I think the addition of the word "random" here doesn't add much. Most of these "create" methods are using random strings!